tvm_linker/src/program.rs

```diff
@@ -199,7 +200,7 @@ pub fn save_to_file(state: StateInit, name: Option<&str>, wc: i8) -> std::result
     let mut print_filename = false;
     let address = state.hash().unwrap();
     let file_name = if name.is_some() {
-        format!("{}.tvc", name.unwrap())
```

Comment:
Can you revert this fix for now? It will break `linker test` command, because `save_to_file` function is used in `call_contract` function with `Some(smc_file)` argument and it is expected that the extension will be added.
Or maybe add exception for test command.

Reply:
Reverted, though it goes against the common practice for `-o` option a little bit. If you could tell me where should I look at the piece of code where this ".tvc" suffix is needed, I will appreciate it.